### ISSUE TITLE
Add an on-screen percentage based on the -exitafter time given

### ISSUE THREAD

**Issue by iammeat** [svenstaro/genact#47]
Thanks for creating such a handy little tool.  It would be great if a percentage could be shown on the screen that correlated with the time given in the -e or -exitafter switches, so that the percentage complete was calculated based on the remaining time until the process will stop.

Maybe it can use a fixed row at the bottom of the terminal, with the output continuing above it?

**Comment by svenstaro** [comment]
Yeah, that sounds pretty cool. Gotta see how to do that in the web version.

**Comment by iammeat** [comment]
Glad you're considering it, and I hope you can find a way to make it work for the web version too.
